[docs] liqoctl generate/delete peering-user (#2957)
liqotech/liqo: docs/usage/peer.md

For the majority and the cases the `liqoctl peer` is enough.
However, **to know the best strategy for each case and the requirements of each approach, check the [peering strategies guide](/advanced/peering-strategies.md)**.

### Establish a peering connection

> To set up a peering with a *provider* cluster, `liqoctl` requires a `kubeconfig` with the necessary permissions to create the resources on the provider cluster for establishing the peering connection.

Reviewer: ```suggestion
To set up a peering with a *provider* cluster, `liqoctl` requires a `kubeconfig` with the necessary permissions to create some resources on the provider cluster, which are used to establish the peering connection.
```

> You can check how to [get a Kubeconfig with the minimum permissions to establish a peering](#get-a-kubeconfig-with-the-minimum-permissions-to-establish-a-peering) with the provider cluster, so preventing any other actions on the other resources of the cluster.

Reviewer: ```suggestion
More details about the specific permissions required in this phase are presented in a dedicated [section](#get-a-kubeconfig-with-the-minimum-permissions-to-establish-a-peering) of the documentation, which shows how this `kubeconfig` prevents any other actions on the other resources of the cluster.
```

```bash
liqoctl --kubeconfig $PROVIDER_KUBECONFIG_PATH unpeer --remote-kubeconfig $CONSUMER_KUBECONFIG_PATH
```

## Get a Kubeconfig with the minimum permissions to establish a peering

The setup of a peering with a *provider* cluster requires the creation of a `kubeconfig` tailored with the proper set of permissions to minimize security risks, namely, to allow the creation of the peering connection with the provider cluster, while preventing any other actions on the other resources on that cluster.

> The [liqoctl](../installation/liqoctl.md) CLI tool provides utility functions to manage the permissions of users able to create a peering connection with the current cluster.

Reviewer: ```suggestion
The [liqoctl](../installation/liqoctl.md) CLI tool provides utility functions to manage the permissions of a dedicate user that is allowed (only) to create a peering connection with the current cluster.
```

```{danger}
liqoctl is designed to support administrators that needs to connect their clusters with Liqo.
Even though **the generated kubeconfig** has the minimum permissions to create a peering connection, it **must never be shared with third parties**, as it still grants some access to the provider cluster.
**In that cases, avoid using liqoctl and refer to the [manual peering](../advanced/manual-peering.md)** guide for a secure setup.
```

**From the *provider* cluster**, you can run the following command to generate a *kubeconfig*:

```bash
liqoctl generate peering-user \
  --kubeconfig $PROVIDER_KUBECONFIG_PATH \
  --consumer-cluster-id $CONSUMER_CLUSTER_ID > $CONSUMER_KUBECONFIG_PATH
```

```{warning}

> Once you generate the *kubeconfig*, take note of it as it will not be stored by Liqo.

Reviewer: ```suggestion
Remember that the generated *kubeconfig* is not saved anywhere by Liqo.
```
